src/utils/isThirdPartyCookieSupported.js

```diff
@@ -10,13 +10,12 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import getBrowser, { CHROME, IE, UNKNOWN } from "./getBrowser";
+import { CHROME, IE, UNKNOWN } from "../constants/browser";
 import includes from "./includes";
 
 // Users could have also disabled third-party cookies in Chrome or IE, but we
 // don't know. We also assume "unknown" browsers support third-party cookies,
 // though we don't really know that either. We're making best guesses.
 const browsersSupportingThirdPartyCookie = [CHROME, IE, UNKNOWN];
 
-export default window =>
-  includes(browsersSupportingThirdPartyCookie, getBrowser(window));
+export default browser => includes(browsersSupportingThirdPartyCookie, browser);
```

Comment:
Notice that `browser` is being passed in rather than this module calling `getBrowser`. Because the browser constants (e.g., `CHROME`, `IE`) are now used here whereas `getBrowser` is not, I felt it important to split the browser constants into their own constants file (`constants/browser.js`) instead of have them as named exports on `utils/getBrowser.js`

Reply:
BTW, does anyone feel like `isThirdPartyCookieSupported` should be renamed to `areThirdPartyCookiesSupported`?